src/Azure.DataApiBuilder.Core/Authorization/AuthorizationResolver.cs

```diff
@@ -209,7 +205,7 @@ public string ProcessDBPolicy(string entityName, string roleName, Config.Operati
         /// <param name="entityName">Entity from request.</param>
         /// <param name="roleName">Role defined in client role header.</param>
         /// <param name="operation">Operation type: create, read, update, delete.</param>
-        /// <returns>Policy string if a policy exists in config.</returns>
+        /// <returns>Policy string if a policy exists in </returns>
```

Comment:
```suggestion
        /// <returns>Policy string if a policy exists in config.</returns>
```

Possible mistake here in changing the comment.